uint8 NUM_ACTUATOR_CONTROLS_ALPHA = 6
uint64 timestamp_sample	    # the timestamp the data this control response is based on was sampled
float32[6] control

# TOPICS voliro_omega
